integration/tests/dialog-multi-select-combo-box.test.js

```diff
@@ -0,0 +1,72 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
+import { sendKeys } from '@web/test-runner-commands';
+import './not-animated-styles.js';
+import '@vaadin/dialog';
+import '@vaadin/multi-select-combo-box';
+
+describe('multi-select-combo-box in dialog', () => {
+  let dialog, comboBox;
+
+  beforeEach(async () => {
+    dialog = fixtureSync('<vaadin-dialog></vaadin-dialog>');
+    dialog.renderer = (root) => {
+      root.innerHTML = '<vaadin-multi-select-combo-box></vaadin-multi-select-combo-box>';
+    };
+    dialog.opened = true;
+    await nextFrame();
+    comboBox = dialog.$.overlay.querySelector('vaadin-multi-select-combo-box');
+    comboBox.items = ['foo', 'bar', 'baz'];
+  });
+
+  afterEach(() => {
+    dialog.opened = false;
+  });
```

Comment:
**nit:** The dialog closes itself in `disconnectedCallback` so manual closing seems to be unnecessary.

Reply:
Thanks, removed.